File: src/main/java/org/funivan/lologa/algo/ga/genome/play/PlayGenerationInterface.java
package org.funivan.lologa.algo.ga.genome.play;

import org.funivan.lologa.algo.ga.genome.GenomeInterface;
import org.funivan.lologa.algo.ga.player.PlayerInterface;

public interface PlayGenerationInterface {
    Iterable<PlayerInterface> play(Iterable<GenomeInterface> genomes);
}

File: src/main/java/org/funivan/lologa/algo/ga/genome/play/PlayGenerationInterface.kt
package org.funivan.lologa.algo.ga.genome.play

import org.funivan.lologa.algo.ga.genome.GenomeInterface
import org.funivan.lologa.algo.ga.player.PlayerInterface

interface PlayGenerationInterface {
    fun play(genomes: Iterable<GenomeInterface>): Iterable<PlayerInterface>
}
